s/babylon/camera/utils/transform-rotate.ts

```diff
@@ -0,0 +1,12 @@
+import {TransformNode} from "@babylonjs/core/Meshes/transformNode.js"
+import {Quaternion} from "@babylonjs/core/Maths/math.vector.js"
+import {v2} from "../../../utils/v2.js"
+export function transformRotate(currentLook: v2.V2, transformA: TransformNode, transformB: TransformNode) {
```

Comment:
maybe a better name like `apply_look_onto_gimbal(look, gimbal)`

and maybe we package up the two transforms and call that a "gimbal" which i believe is accurate, or somewhat accurate

```ts
export type Gimbal = {
  a: TransformNode
  b: TransformNode
}
```